content/src/main/content/jcr_root/apps/acs-commons/authoring/sitewide-component-props/sitewide-component-props.js

```diff
@@ -0,0 +1,55 @@
+(function ($, ns, channel, window, undefined) {
+
+    var actionDef = {
+        icon: 'coral-Icon--globe',
+        text: Granite.I18n.get('Configure Shared Content'),
+        handler: function (editable, param, target) { // will be called on click
+        	var originalDialogSrc = editable.config.dialogSrc;
+        	var originalDialog = editable.config.dialog;
+        	
+        	var dialogSrcArray = editable.config.dialogSrc.split(".html");
+        	var langRootRegexp = /^(\/content\/([^/]+\/)+([a-zA-Z]{2}([_-][a-zA-Z]{2})?))(|\/.*)$/g;
+        	var match = langRootRegexp.exec(dialogSrcArray[1]);
+        	
+        	var siteWideDialogSrc = dialogSrcArray[0].replace("_cq_dialog", "_cq_dialogsitewide") +
+						        	".html" +
+						        	match[1] +
+						        	"/jcr:content/sitewideprops/" +
+						        	editable.type;
+        	
+        	editable.config.dialogSrc = siteWideDialogSrc;
+        	editable.config.dialog = editable.config.dialog.replace("cq:dialog", "cq:dialogsitewide");
+        	
+        	ns.edit.actions.doConfigure(editable);
+        	
+        	//set the dialog and dialogSrc back to the original values so normal edit dialog continues to work
+        	editable.config.dialogSrc = originalDialogSrc;
```

Comment:
Wonder if we should put these in a `finally` block for a try/catch, to make absolute sure this happens.